action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   environment-variables:
     description: 'Variables to add to the container. Each variable is of the form KEY=value, you can specify multiple variables with multi-line YAML strings.'
     required: false
+  docker-lables:
```

Comment:
This should be `docker-labels:`, not `docker-lables:`.

Reply:
thanks you b70f4ff